srcpkgs/minecraft/template

```diff
@@ -1,15 +1,26 @@
 # Template file for 'minecraft'
 pkgname=minecraft
-version=0.5
+version=2.1.5965
 revision=1
 archs=noarch
-depends="wmname wget"
+wrksrc=${pkgname}-launcher
+depends="virtual?java-runtime GConf alsa-lib gtk+ libX11 libXScrnSaver libXtst libxcb nss xrandr"
 short_desc="Launcher for the famous Minecraft game"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="Matthew Treadwell <hewtreadwell@gmail.com>"
+license="custom:Minecraft"
 homepage="http://www.minecraft.net/"
-license="Propietary"
+distfiles="https://launcher.mojang.com/download/linux/x86_64/minecraft-launcher_${version}.tar.gz"
+checksum=85a0a2478861c3e660a1273f6b6ac2e14232ae439cd6d6172391065274228c72
 repository=nonfree
```

Comment:
License forbids redistribution, so need to set restricted=yes, what means it won't go to repo: one have to build package for oneself.